docs/docs/testing.md

```
@@ -429,3 +432,22 @@ describe("SomeIntegrationTestWithDB", () => {
   afterAll(PlatformTest.reset);
 });
 ```
+
+## Use TestContainers <Badge text="v8.9.0+" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: Badge

(MD033, no-inline-html)